code/game/objects/effects/effect_system/effects_sparks.dm

```diff
@@ -28,20 +28,20 @@
 	playsound(src.loc, "sparks", 100, 1)
 	var/turf/T = loc
 	if(isturf(T))
-		T.hotspot_expose(1000,100)
+		T.hotspot_expose(1000,100,0,1)
```

Comment:
, FALSE, TRUE

Reply:
Wait is there supposed to be a space before the false/true?